src/ziggurat/streams.clj

```diff
@@ -208,6 +208,9 @@
      :cfg    cfg-2}))
 
 (defn- stream-joins-topology [handler-fn {:keys [input-topics join-cfg oldest-processed-message-in-s]} topic-entity channels]
+  (log/warn "[Alpha Feature]: Stream joins is an alpha feature."
```

Comment:
this might be a stretch, given that this only shows on stdout, people might not notice this when they ran the actor. can we somehow colorize the output and show a bigger ascii text? like I said this might be a stretch...